src/test/resources/createDB.sql

```diff
@@ -26,3 +26,9 @@ CREATE TABLE TransactionHistory (
   Action varchar(255) CHECK (Action IN ('Deposit', 'Withdraw')),
```

Comment:
@jacktzhao and I will need to add `Transfer` as a valid action here as per the new re-design.